.azure/azure-pipelines.ci.yml

```diff
@@ -209,13 +209,14 @@ stages:
       arch: x64
       tls: openssl
       extraBuildArgs: -DisableTools -DisableTest -EnableTelemetryAsserts
-  - template: ./templates/build-config-user.yml
-    parameters:
-      image: windows-latest
-      platform: gamecore
-      arch: x64
-      tls: schannel
-      extraBuildArgs: -DisableTools -DisableTest -EnableTelemetryAsserts
+  # Gamecore will not be able to be built in public CI
```

Comment:
Is the because the GDK isn't available? If so, then https://github.com/actions/virtual-environments/issues/4053 should unblock that (eventually).

Reply:
It will never work because it requires private gamecore.

Reply:
If this doesn't work, just delete it instead of commenting it out.